<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainW</class>
 <widget class="QWidget" name="MainW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>control</string>
  </property>
  <widget class="QPushButton" name="selectMasterButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>select master</string>
   </property>
  </widget>
  <widget class="QLabel" name="masterPathLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>path</string>
   </property>
  </widget>
  <widget class="QLabel" name="rat1IDLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>59</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;rat1 ID&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="rat2IDLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>59</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;rat2 ID&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rat1IDInput">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="rat2IDInput">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>84</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="videoDurationBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="value">
    <number>60</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="numVideosBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="videoDurationLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>169</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;duration per Video&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="numVideosLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;number of videos&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="videoDurationUnitLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;minutes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>84</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainW</receiver>
   <slot>quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectMasterButton</sender>
   <signal>clicked()</signal>
   <receiver>MainW</receiver>
   <slot>selectMasterFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>MainW</receiver>
   <slot>startRun()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>MainW</receiver>
   <slot>stopRun()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>quit()</slot>
  <slot>selectMasterFolder()</slot>
  <slot>startRun()</slot>
  <slot>stopRun()</slot>
 </slots>
</ui>
